packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -215,6 +226,7 @@ Typing into the filter bar filters the list options in the dropdown (not picture
 | `HOME`                 | Opens the dropdown and moves visual focus to the first list option                                                                                                                      |
 | `END`                  | Opens the dropdown and moves visual focus to the last list option                                                                                                                       |
 | _Printable characters_ | Opens the dropdown (if closed), moves visual focus to the filter input (if filterable) with the characters typed or to the first list option that matches the typed character or string |
+| `ESC`                  | Clears the selected item without moving focus (if clearable)                                                                                                                            |
```

Comment:
There is very little consensus about the keyboard interactions in existing design systems:
1. The [majority](https://shoelace.style/components/select#clearable) of select/dropdown controls (that allow clearing the selected option) only support the 'clear selection button'. None of them support tab focus of the 'clear selection button'. 
2. A small [minority](https://dash.plotly.com/dash-core-components/dropdown) support `ESC`
3. A small [minority](https://style.monday.com/?path=/story/inputs-dropdown--dropdown-value-selection) support `Delete`.

| Approach | Pros | Cons |
| --- | ---| ----|
| Button only | Majority rule | Not keyboard accessible |
| `ESC` | Consistent with Combobox [ARIA guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) | Also used to close dropdown, and some users expect to use it to close slide-in dialogs |
| 'Delete` | Consistent with likely future multi-select/picker keyboard interactions | Low discoverability |

Reply:
`ESC` feels natural enough to me and the alignment with the ARIA combobox pattern is compelling. I don't think the cons are dealbreakers.

Reply:
Maybe a mitigation for the con is that while the select is focused it always is capturing `esc`?
I can't remember if that would be in conflict with the future close watcher changes @m-akinc (but maybe we cross that bridge when we get to it)